language/sygus.tex

```diff
@@ -293,6 +299,11 @@ \subsubsection{Changes from SyGuS 2.0 to SyGuS 2.1}
 \item Oracle Functions are introduced. These are functional symbols whose interpretation is associated to an external oracle. The functions are treated as universally quantified uninterpreted functions, and calls to that external oracle place with concrete input values generate assumptions over the behaviour of the uninterpreted function. These assumptions are conjoined to the list of assumptions $\alpha$ in the synthesis conjecture.
 
 \item A set of new commands are introduced to enable easy specification of specific types of oracles, for instance input-output oracles. These oracles are based on the formalization by Jha and Seshia~\cite{oracles}.
+
+\item A new SyGuS logic, called \sygusinline|CHC_$X$|,
+has been introduced for synthesis problems over constrained Horn clause systems over an SMT-LIB base logic $X$.
+
+\item The $\synthinvkwd$ command has been renamed to $\synthpredkwd$ command.
```

Comment:
Is there actually any need to have `synth-pred`? Technically, a "predicate" is just a Boolean function.  Can we perhaps just omit this distinction and always use `synth-fun` (where you must supply the Boolean return type)?

Reply:
I would be okay with this change. Writing a `synth-pred` in place of `synth-fun` only saves us 4 bytes haha.

I can update Section 5.5 if we all agree.

What do you think @polgreen?

Reply:
sounds fine to me

Reply:
Updated the doc as per this discussion.